src/PKSim.Infrastructure/ORM/Repositories/OntogenyRepository.cs

```diff
@@ -115,6 +184,29 @@ private Ontogeny forMolecule(OriginData originData, string molecule)
          return allFor(originData.Species.Name).FindByName(molecule) ?? new NullOntogeny();
       }
 
+      public DataRepository OntogenyToRepository(Ontogeny ontogeny, string containerName)
+      {
+         var dataRepository = new DataRepository {Name = PKSimConstants.UI.OntogenyFor(ontogeny.DisplayName, containerName)};
+         var xUnit = _displayUnitRetriever.PreferredUnitFor(_dimensionRepository.AgeInWeeks);
```

Comment:
Why "Age in weeks" and not "Age in years" here?

Reply:
Indeed. I fixed in already in another PR